Package.resolved

```diff
@@ -15,8 +15,17 @@
         "repositoryURL": "https://github.com/daltoniam/Starscream.git",
         "state": {
           "branch": null,
-          "revision": "9c03ef715d1bc9334b446c90df53586dd38cf849",
-          "version": "3.1.0"
+          "revision": "e6b65c6d9077ea48b4a7bdda8994a1d3c6969c8d",
+          "version": "3.1.1"
```

Comment:
on your machine, if you remove the Package.resolved, does it automatically resolves to the 3.1.1? it doesn't on my computer.

Reply:
Yes, because I did a `swift package update` and dependency is declared as `.upToNextMinor`. There seems to be a new release of StarScream (https://github.com/daltoniam/Starscream/releases). We can revert the change but it seems safe.

Reply:
good 👍